src/Components/Checkout.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+class Checkout extends React.Component() {
```

Comment:
No parens necessary here - they are causing an error!
What's the difference between React.Component and React.Component()?
You may want to review how to declare react class components and ES6 class syntax.